staging/src/k8s.io/legacy-cloud-providers/gce/gce.go

```diff
@@ -659,6 +659,11 @@ func (g *Cloud) Instances() (cloudprovider.Instances, bool) {
 	return g, true
 }
 
+// InstancesV2 returns an implementation of InstancesV2 for Google Compute Engine.
+func (g *Cloud) InstancesV2() (cloudprovider.InstancesV2, bool) {
+	return g, true
```

Comment:
I don't believe the gcp cloud provider should be returning true as it does not yet implement InstanceShutdownByProviderID. (https://github.com/kubernetes/kubernetes/blob/440b5c2fbc82f4bba3f5d3653ffbd77dec481aa0/staging/src/k8s.io/legacy-cloud-providers/gce/gce_instances.go#L174) Have we verified that the rest of the in-tree providers actually fully implement the interface?

Reply:
> I don't believe the gcp cloud provider should be returning true as it does not yet implement InstanceShutdownByProviderID.

It's consistent with Instances interface.

> Have we verified that the rest of the in-tree providers actually fully implement the interface?

Yes, others in-tree providers all implement InstanceShutdownByProviderID.